composer.json

```diff
@@ -14,16 +14,17 @@
     "require": {
         "php": "^5.5.9|^7.0",
         "symfony/framework-bundle": "^2.8|^3.0",
-        "symfony-cmf/slugifier-api": "^1.0"
+        "symfony-cmf/slugifier-api": "^1.0",
+        "symfony/symfony": "3.1.*"
```

Comment:
this should not be in the composer.json file. we do it dynamically in .travis.yml to test specific symfony versions. but for installation purposes we depend on framework-bundle